### ISSUE TITLE
docker run bash -c "some parameters" only read first parameter

### ISSUE THREAD

**Issue by tdeheurles** [docker/toolbox#368]
Hey,

I've just got an issue running that code:

```bash
docker run ubuntu:14.04.2 bash -c "mkdir /foo"
mkdir: missing operand
Try 'mkdir --help' for more information.
```

It seems that the second parameter is not seen anymore.

Another example:

**ubuntu:14.04.2**
```bash
$ docker run ubuntu:14.04.2 bash -c "ls"
bin
boot
dev
[...]
```

```bash
$ docker run ubuntu:14.04.2 bash -c "ls /root"
bin
boot
dev
[...]
```

**BusyBox v1.24.1**
```bash
$ docker run busybox sh -c 'mkdir /foo'
Usage: mkdir [OPTIONS] DIRECTORY...

Create DIRECTORY

        -m MODE Mode
        -p      No error if exists; make parent directories as needed

```
This example is with:
- DockerToolbox-1.9.1h
- GNU bash, version 4.3.39(3)-release (x86_64-pc-msys)

**Comment by tdeheurles** [comment]
The issue is still present in docker `1.9.1i`

**Comment by tdeheurles** [comment]
Still the same with `docker 1.10 release`:
```
$ docker version
Client:
 Version:      1.10.0
 API version:  1.22
 Go version:   go1.5.3
 Git commit:   590d5108
 Built:        Thu Feb  4 18:18:11 2016
 OS/Arch:      windows/amd64

Server:
 Version:      1.10.0
 API version:  1.22
 Go version:   go1.5.3
 Git commit:   590d5108
 Built:        Thu Feb  4 19:55:25 2016
 OS/Arch:      linux/amd64

$ docker run ubuntu:14.04.2 bash -c "mkdir /tmp/test"
mkdir: missing operand
Try 'mkdir --help' for more information.
```

**Comment by Bakudankun** [comment]
Changing `$@` in [start.sh](https://github.com/docker/toolbox/blob/master/windows/start.sh#L62) to `"$@"` may fix this bug.

**Comment by tdeheurles** [comment]
docker command is a function in last version in order to add `MSYS_NO_PATHCONV=1`

Running docker toolbox as a function in version 1.9.1g duplicate the bug. It seems that you are right @Bakudankun. I'll test with the newer version. Thank you.

**Comment by tdeheurles** [comment]
@Bakudankun : confirmed, it works with 1.10 too. Thank you.

**Comment by jeffdm** [comment]
Fixed in #423 - closing this!

Thanks again all!
